.circleci/config.yml

```diff
@@ -0,0 +1,156 @@
+---
+version: 2.1
+executors:
+  med: # 2cpu, 4G ram
+    docker:
+      - image: circleci/node:10
+    resource_class: medium
+    working_directory: ~/project
+    environment:
+      GRADLE_OPTS: -Xmx2048m -Dorg.gradle.daemon=false -Dorg.gradle.parallel=true -Dorg.gradle.workers.max=2
```

Comment:
Do we need this line for a node build? Or are there java deps in the tests which will use this?

Reply:
Just bad copy-paste, now fixed